KingfisherTests/ImageCacheTests.swift

```diff
@@ -187,7 +187,11 @@ class ImageCacheTests: XCTestCase {
 
                 XCTAssert(noti.object === self.cache, "The object of notification should be the cache object.")
                 
-                let hashes = noti.userInfo?[KingfisherDiskCacheCleanedHashKey] as! [String]
+                guard let hashes = noti.userInfo?[KingfisherDiskCacheCleanedHashKey] as? [String] else {
+                    XCTFail("The clean disk cache notification should contains Strings in key 'KingfisherDiskCacheCleanedHashKey'")
```

Comment:
Length Violation: Line should be 100 characters or less: currently 131 characters